package.json

```diff
@@ -69,5 +69,8 @@
   "homepage": "https://github.com/freestarcapital/freestar-plugin-react-native",
   "author": {
     "name": "Freestar"
+  },
+  "dependencies": {
+    "@freestar/freestar-plugin-react-native": "github:freestarcapital/freestar-plugin-react-native"
```

Comment:
@carlosalcala-freestar i could be wrong here, but your dependency is referencing itself, so that's potentially a circular dependency problem.

Reply:
Fixed